docs/concepts/gardenlet.md

```diff
@@ -35,13 +35,28 @@ After you depoyed it yourself into your seed clusters, it initializes a bootstra
 * Gardenlet starts up with a bootstrap kubeconfig having a bootstrap token that allows to create `CertificateSigningRequest` resources
 * After the CSR is signed it downloads the created client certificate, creates a new kubeconfig with it, and stores it inside a `Secret` in the seed cluster
 * It deletes the bootstrap kubeconfig secret and starts up with its new kubeconfig
+* Gardenlet starts normal operation
 
 Basically, you can follow the Kubernetes documentation regarding the process.
 The gardener-controller-manager runs a control loop that automatically signs CSRs created by Gardenlets.
 
-:warning: Certificate rotation is not yet implemented but will follow very soon.
+Optionally, if you don't want to run this bootstrap process, then you can create a kubeconfig pointing to the Garden cluster for the Gardenlet yourself, and simply provide it to it (field `gardenClientConnection.kubeconfig` in the Gardenlet configuration).
 
-Optionally, if you don't want to run this bootstrap process, then you can create a Kubeconfig pointing to the Garden cluster for the Gardenlet yourself, and simply provide it to it.
+### Gardenlet certificate rotation
+
+The Gardenlet tries to automatically rotate the certificate when it approaches expiration of its one year lifetime (at ~80% already passed).
+In order to use certificate rotation, the Gardenlet [component configuration](#component-configuration) needs to set the field `.gardenClientConnection.kubeconfigSecret`, specifying the secret to store the kubeconfig with the rotated certificate.
```

Comment:
```suggestion
In order to use certificate rotation, the Gardenlet [component configuration](#component-configuration) needs to set the field `.gardenClientConnection.kubeconfigSecret`, specifying the secret to store the kubeconfig with the certificate to be rotated.
```

Reply:
the docu is correct :) It will always save the rotated cert in the secret, but the secret to be rotated doe not necessarely have to be stored in the secret alread. Can also rotate the locally provided `gardenCLientConnection.kubeconfig`.